assets/src/components/form/select/model-select.js

```diff
@@ -133,7 +130,7 @@ export class ModelSelect extends Component {
 
 	getSelectLabel() {
 		const { selectLabel, selectConfiguration } = this.props;
-		return selectLabel ?
+		return selectLabel !== MODEL_SELECT_LABEL_NONE ?
```

Comment:
added this when I added the `EditorSelect` component which utilizes the WordPress core `BaseControl` component for adding input labels in the admin. 
Initially I had tried unsetting (or setting to empty) the label attribute for the wrapped select component from the specific `EditorSelect` (ie: `DatetimeEditorSelect` would override  the label set in `DatetimeSelect`) but that did not work and `ModelSelect` would still display the actual html label tag, but just with no text in it.

I can try it again without using a constant as maybe I was doing something incorrectly.

Reply:
This should not be necessary.  An falsey value for the `selectLabel` prop should be enough to prevent the label from being generated.  I don't like the fact that ANY value other than `MODEL_SELECT_LABEL_NONE` will result in the label being generated.